com.unity.render-pipelines.high-definition/Runtime/Material/BuiltinGIUtilities.hlsl

```diff
@@ -83,14 +86,8 @@ void EvaluateLightProbeBuiltin(float3 positionRWS, float3 normalWS, float3 backN
     }
     else
     {
-#if RAYTRACING_ENABLED
-        if (unity_ProbeVolumeParams.w == 1.0)
-            SampleProbeVolumeSH9(TEXTURE3D_ARGS(unity_ProbeVolumeSH, samplerunity_ProbeVolumeSH), positionRWS, normalWS, backNormalWS, GetProbeVolumeWorldToObject(),
```

Comment:
Is it intetionnal that the SH9 mode just got removed here?

Reply:
Yes, not used anymore